### DISCUSSION TITLE
Proper way to revert to a previous version?

### DISCUSSION THREAD

**Discussion by CartierCodes** [Rem0o/FanControl.Releases#2605] [Q&A]
Just wondering what the intended way to rollback to the previous version is. V195 broke my setup so looking to go back to V194. Obviously want to keep my settings if possible.

**Comment by Rem0o** [comment]
You can use any release from https://github.com/Rem0o/FanControl.Releases/releases.

However, I would like to know how the new version broke your config?

**Reply by CartierCodes** [reply]
The sensors related to ADLX are misnamed in my new version. The GPU sensors are labeled integrated graphics related and vice versa. I believe this because the GPU sensors are showing 0s and the integrated graphics sensors are showing the temps I am used to seeing with my GPU

**Reply by ElliotThiebaut** [reply]
@Rem0o Just enabled ADLX and had the same issue as @CartierCodes, my 7900XTX shows 0 degrees on GPU and hotspot but my integrated graphics (from my 7800X3D) shows temps (40C and 50C), not sure if they correspond to my dedicated GPU temps or something else.

**Reply by TheRealRedDragon** [reply]
The ADLX code doesn't seem to work.  Whenever I start playing a game, my GPU is running at 100%, it's ignoring the curve I created, and the app no longer is able to control fan speeds.  If I apply a 0% fan curve it has no effect, same if I apply a 100% fan curve.  I'm reverting to 1.94 as well.  FYI, all my other sensors work, and I can control all of those fans manually, just can't control the GPU fans.
